docs/detections/alerts-view-details.asciidoc

```diff
@@ -6,111 +6,159 @@
-* *Alerts related by process ancestry* - Shows alerts that are related by process events on the same linear branch. Note that alerts generated from processes on child or related branches are not shown. To further examine alerts, click *Investigate in timeline*.
+* **MITRE ATT&CK**: Provides relevant https://attack.mitre.org/[MITRE ATT&CK] framework tactics, techniques, and sub-techniques.
 
 [discrete]
-[[alerts-enrich-host-user-risk-score]]
-==== Alerts enriched with user and host risk scores
-Alerts can be enriched with user and host risk scores, which convey the level of risk associated with a specific user and host. Risk levels are `Unknown`, `Low`, `Moderate`, `High`, or `Critical`. Refer to <<host-risk-score>> and <<user-risk-score>> to learn how risk scores are calculated.
+[[investigation-section]]
+== Investigation 
 
-NOTE: User and host risk scores are technical preview features and require a https://www.elastic.co/pricing[Platinum subscription] or higher. You must enable user and host risk score features to access risk scores data. Refer to <<host-risk-score>> and <<user-risk-score>> to learn more.
+The Investigation section is located on the **Overview** tab in the right panel. It offers a couple of ways to begin investigating the alert. 
 
 [role="screenshot"]
-image::images/enriched-host-user-rs.png[Host and user risk score subsections with risk scores, 600]
+image::images/investigation-section-rp.png[Investigation section of the Overview tab, 65%]
 
-Two types of user and host risk scores can appear in the Enriched data section:
+The Investigation section provides the following information:
 
-* **Current user/host risk classification:** The current risk score of the user or host associated with the alert.
-* **Original user/host risk classification:** The first risk score that was calculated for the user or host associated with the alert.
+* **Investigation guide**: The **Show investigation guide** button displays if the rule associated with the alert has an investigation guide. Click the button to open the **Investigation** tab in the left panel of the alert details flyout.  
 +
-The *Original user/host risk classification* field only displays if the current risk score no longer matches the original risk score. In this situation, both the current and original risk scores appear, showing how the risk changed.
+TIP: Add an <<add-ig-actions-rule,investigation guide>> to a rule when creating a new custom rule or modifying an existing custom rule's settings.
+
+* **Highlighted fields**: Shows relevant fields for the alert and any custom highlighted fields you added to the rule. 
```

Comment:
@yctercero just an fyi that I doc'd the custom highlighted fields functionality here too.